clients/java/src/main/java/io/unitycatalog/client/retry/FixedDelayRetryPolicy.java

```diff
@@ -0,0 +1,51 @@
+package io.unitycatalog.client.retry;
+
+import java.time.Duration;
+
+public class FixedDelayRetryPolicy implements RetryPolicy {
```

Comment:
I just keep it here for another kind of implementation for `RetryPolicy`, although we did not really use it in the production code.